.github/workflows/build-on-windows.yml

```diff
@@ -0,0 +1,29 @@
+name: Build gProfiler on Windows
+
+on: pull_request
+# on:
+#   pull_request:
+#   push:
+#     tags:
+#       - 'windows'
+
+jobs:
+  build-executable:
+   runs-on: windows-latest
+   steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: install chocolatey
+        run: Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+      
+      - name: install wget
```

Comment:
Shouldn't need to install `wget`, github workers should already support `curl` [as it comes with Windows](https://learn.microsoft.com/en-us/virtualization/community/team-blog/2017/20171219-tar-and-curl-come-to-windows).
Use `curl` instead.